workflows/misc_file_types/hobo_files.Rmd

```diff
@@ -0,0 +1,12 @@
+---
+title: "hobo-files"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
```

Comment:
To fit in with the formatting of our bookdown, you'll have to make a few quick changes: 

- use `##` for the header, and maybe retitle it something more general: `Datalogger files` ([example](https://github.com/NCEAS/datateam-training/edit/master/workflows/misc_file_types/spatial_data.Rmd))
- no need to include the `{r setup}...` chunk, since you're not providing code snippets anyway